core/src/main/java/org/lflang/generator/c/CCompiler.java

```diff
@@ -370,6 +370,7 @@ private boolean outputContainsKnownCMakeErrors(String CMakeOutput) {
    *     the compile command will produce a binary.
    */
   public LFCommand compileCCommand(String fileToCompile, boolean noBinary) {
+    // FIXME: This is obsolete.
```

Comment:
We should address this or tie it to an issue.

Reply:
This again has nothing to do with this PR, but I've removed the obsolete function in commit #8fbdd58cd.